cssom-view-1/Overview.bs

```diff
@@ -783,22 +783,24 @@ the associated <a>media</a>.
 The <dfn attribute for=MediaQueryList>matches</dfn> attribute must return
 the associated <a>matches state</a>. <!--fingerprint-->
 
-The <dfn method for=MediaQueryList>addListener(<var>listener</var>)</dfn> method must run these steps:
+The <dfn method for=MediaQueryList>addListener(<var>callback</var>)</dfn> method,
+when invoked, must run these steps:
 
-1. If <var>listener</var> is null, terminate these steps.
-1. Append an <a>event listener</a> to the associated list of <a>event listeners</a>
-    with <b>type</b> set to <code>change</code>,
-    <b>callback</b> set to <var>listener</var>,
-    and <b>capture</b> set to false,
-    unless there already is an <a>event listener</a> in that list
-    with the same <b>type</b>, <b>callback</b>, and <b>capture</b>.
+1. <a spec=dom>Add an event listener</a> with the <a spec=dom>context object</a>
+    and an <a spec=dom>event listener</a> whose
+    <a spec=dom for="event listener">type</a> is <code>change</code>,
+    and <a spec=dom for="event listener">callback</a> is |callback|.
```

Comment:
This should probably specify what happens when `callback` is null explicitly, which is nothing?

Though I guess per https://dom.spec.whatwg.org/#add-an-event-listener step 2 we're alright, so maybe not worth it.

Reply:
Yeah, no need to check it multiple times.